src/quo2/components/share/share_qr_code/view.cljs

```diff
@@ -0,0 +1,49 @@
+(ns quo2.components.share.share-qr-code.view
+  (:require [quo2.components.share.share-qr-code.style :as style]
+            [quo2.components.share.qr-code.view :as qr-code]
+            [react-native.blur :as blur]
+            [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.buttons.button :as button]
+            [react-native.core :as rn]))
+
+(defn view
+  [{:keys [source link-title
+           url-on-press url-on-long-press qr-url share-on-press]}]
+  [blur/ios-view
```

Comment:
@ulisesmac @smohamedjavid @ilmotta so it was the blur view in the actual component (not the preview screen) causing the issue.

This `blur/ios-view` is blur for ios and rn/view for android.

Now it looks like this:
<img width="289" alt="Screenshot 2023-04-05 at 20 55 19" src="https://user-images.githubusercontent.com/22799766/230197216-b792db93-6b8c-40cd-8558-3c033df0db2e.png">

Perhaps we also need to adjust the blur preview component too.
we can try different approaches ofc :)